includes/blocks/components/class-progress-bar.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * File containing the Progress_Bar class.
+ *
+ * @package sensei
+ * @since   4.0.0
+ */
+
+namespace Sensei\Blocks\Components;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Common progress bar block component, with a label.
+ */
+class Progress_Bar {
+
+	/**
+	 * Renders the progress bar component.
+	 *
+	 * @access private
+	 *
+	 * @param array $attributes {
+	 *                          Component attributes.
+	 *
+	 * @type string $label      Text label.
```

Comment:
This indentation seems a little weird. Adding spaces before the `@type` would work? Or would be a PHPCS issue?

Reply:
Aligned it in [66cbeb8](https://github.com/Automattic/sensei/pull/4668/commits/66cbeb8f37a556236157eaf230fd45e70ded3756#diff-92a9a32d8c58f2b4a4bf94fa70feaf2704527f6c23216e050498135e13bebca0)